ddtrace/contrib/django/apps.py

```diff
@@ -68,3 +69,12 @@ def ready(self):
                     patch_cache(tracer)
                 except Exception:
                     log.exception('error patching Django cache')
+
+            # Instrument rest_framework app to trace custom exception handling.
+            # rest_framework only supports django 1.10, 1.11 and 2.0.
+            if apps.is_installed('rest_framework') and django.VERSION >= (1, 10):
```

Comment:
I think we don't need the `and django.VERSION >= (1, 10)` clause, if Django 1.8 is used with an incompatible version of DRF, neither the user app will work so we can assume here that it's properly configured. Let's have these check only if we do a "special" import.